frappe/templates/includes/web_sidebar.html

```diff
@@ -37,7 +37,7 @@ <h6>{{ item.group_title }}</h6>
 {% macro my_account() %}
 {% if frappe.user != 'Guest' %}
 <ul class="list-unstyled">
-	<li class="sidebar-item">
+	<li class="sidebar-item" style="background-color: #D1E8FF;">
```

Comment:
Where did you get this color from? Is there a variable that we can use?
Do not use inline styling .. you can modify the scss instead